http/http_rest_proxy.cc

```diff
@@ -306,7 +308,8 @@ perform(const std::string & verb,
         bool exceptions,
         OnData onData,
         OnHeader onHeader,
-        bool followRedirect) const
+        bool followRedirect,
+        bool arbitraryTooSlowAbort) const
```

Comment:
Also, we should use the same name everywhere, some are prefixed with http some are not.